lib/components/helpers/pretty-text-input.js

```diff
@@ -283,7 +283,7 @@ export default React.createClass({
     return (
       <div onKeyDown={this.onKeyDown} className={cx({'pretty-text-wrapper': true, 'choices-open': this.state.isChoicesOpen})}
            onMouseEnter={this.switchToCodeMirror} onTouchStart={this.switchToCodeMirror}>
-        <div onClick={this.onFocusClick}>
+        <div className="sub-pretty-text-wrapper" onClick={this.onFocusClick}>
```

Comment:
That's a pretty funny name. Maybe `pretty-text-click-wrapper`? Still a funny name, but maybe a bit more descriptive. BTW, reason for this `div` here: https://github.com/zapier/formatic/commit/cbf9680a56e94fa5c3251c4d23dc95bfd2d496be#commitcomment-20264080.